adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -121,6 +121,121 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] {
     replaceRdd(tFn(rdd))
   }
 
+  private[rdd] val sortedTrait: SortedTrait
+  private[rdd] def sorted = sortedTrait.sorted
+  private[rdd] def partitionMap = sortedTrait.partitionMap
+
+  /**
+   * This case class is the source of all sorted knowledge handling in ADAM. We
```

Comment:
`sorted` doesn't need to be abstract, the GenomicRDD is either sorted or it ain't.  And an empty `partitionMap` is probably ok too.